capellambse/extensions/metrics/collector.py

```diff
@@ -9,7 +9,21 @@
 modeling focus is (problem space / solution space / balanced)
 """
 
+from __future__ import annotations
+
+__all__ = [
+    "BinCountResults",
+    "get_compliance_score",
+    "get_passed_and_total",
+    "get_total_objects",
+    "quantify_model_layers",
+]
+
+import collections.abc as cabc
+from itertools import chain
+
 import capellambse
+from capellambse.extensions import validation as v
```

Comment:
Honestly I don't think `validation` is "an inconveniently long name" that warrants being abbreviated. It's only used on 5 lines in this file, and single letter abbreviations can be very annoying to grep for. We should keep this reasonable.

Reply:
cannot agree more